pom.xml

```diff
@@ -0,0 +1,364 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
```

Comment:
Please remove the copyright notice, we try not to add those to pom files as they're not necessary

Reply:
Will do.

Reply:
Done